ui/src/app/data-explorer/services/data-explorer-field-provider-service.ts

```diff
@@ -194,10 +194,12 @@ export class DataExplorerFieldProviderService {
                 runtimeType === 'http://www.w3.org/2001/XMLSchema#float' ||
```

Comment:
I probably missed something here, but why do we have duplicate conditions here? line 193 until 197 is the same as 198 to 202, isn't it?

Reply:
The string prefix `https://www.w3.org/2001` appears only five times in all files, and it only appears in files related to the ui. It seems that we can remove the code associated with the prefix...

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/30497621/220693506-afd59b32-01ab-402d-ba2c-52e3e48ec0fb.png">

Reply:
Ah it is about `http` vs `https` 💡 
Didn't recognize the decisive difference so far 😅

Reply:
Haha, yes it's about http vs https 😂

But it's weird we have those https in our code, it seems to be useless... I have pushed a commit to remove them.